src/registrar/forms/application_wizard.py

```diff
@@ -113,5 +116,34 @@ def get_context_data(self, form, **kwargs):
         context["form_titles"] = TITLES
         return context
 
-    def done(self, form_list, **kwargs):
+    def forms_to_object(self, form_dict: dict) -> DomainApplication:
+        """Unpack the form responses onto the model object properties."""
+        application = DomainApplication.objects.create(creator=self.request.user)
+
+        # organization information
+        organization_data = form_dict["organization"].cleaned_data
+        application.organization_type = organization_data["organization_type"]
+        application.federal_branch = organization_data["federal_type"]
+        application.is_election_office = organization_data["is_election_board"]
+
+        # contact information
+        contact_data = form_dict["contact"].cleaned_data
+        application.organization_name = contact_data["organization_name"]
+        application.street_address = contact_data["street_address"]
+        # TODO: add the rest of these fields when they are created in the forms
+
+        # This isn't really the requested_domain field
+        # but we need something in this field to make the form submittable
+        requested_site, _ = Website.objects.get_or_create(
+            website=contact_data["organization_name"] + ".gov"
+        )
+        application.requested_domain = requested_site
+        return application
+
+    def done(self, form_list, form_dict, **kwargs):
         logger.info("Application form submitted.")
+        application = self.forms_to_object(form_dict)
+        application.submit()  # change the status to submitted
```

Comment:
**Suggestion:** Seeing this function used now in its natural state and how the naming of states could end up being quite confusing, I wonder if we'd do better with a setter method to wrap the actual transition methods. Not a real Python setter (unless you want), just something like `application.set_status(to=DomainApplication.SUBMITTED)`.